src/actions/index.js

```diff
@@ -42,8 +45,27 @@ export const completeTodo = (id, date) => {
     id,
     isTaskComplete: true,
     completeTime: getDays(date.getTime(), new Date().getTime()),
+    isEditBtnClicked: false,
   };
 };
+
+export const editTodo = (id, editedData) => {
```

Comment:
Could we be more explicit here naming arguments ?
```suggestion
export const editTodo = (id, editedTask) => {
```